astroquery/mast/cutouts.py

```diff
@@ -408,14 +441,25 @@ def get_cutouts(self, *, coordinates=None, size=5, product='SPOC', sector=None,
             first majorbody is tried and then smallbody if a matching majorbody is not found.
 
             NOTE: If moving_target is supplied, this argument is ignored.
+        timeout_add : int or float, optional
+            The amount (in seconds) by which the request processing time upper limit will be changed.
```

Comment:
Yes, I've changed the `_change_timeout` method now to `_get_default_timeout` so that the `TIMEOUT` attribute gets written back to the default if `timeout` is called.